lib/features/pay/pages/pay_page.dart

```diff
@@ -59,7 +62,10 @@ class PayPageState extends State<PayPage> {
                 const SizedBox(height: 20),
                 Text(
                   '£$amount.00',
-                  style: const TextStyle(fontSize: 48, color: Colors.white, fontWeight: FontWeight.bold),
+                  style: const TextStyle(
+                      fontSize: 48,
+                      color: Colors.white,
+                      fontWeight: FontWeight.bold),
```

Comment:
Pu comma at the end of each list of parameters where it is not possible to fit them all in one line